package.json

```diff
@@ -66,7 +66,8 @@
     "prettier-check": "prettier -c --ignore-path .gitignore \"**/*.{css,html,js,json,md,yaml,yml}\"",
     "prettier-check-updated": "pipe-git-updated --ext=css --ext=html --ext=js --ext=json --ext=md --ext=yaml --ext=yml -- prettier -c",
     "prettify": "prettier --write --ignore-path .gitignore \"**/*.{css,html,js,json,md,yaml,yml}\"",
-    "prettify-updated": "pipe-git-updated --ext=css --ext=html --ext=js --ext=json --ext=md --ext=yaml --ext=yml -- prettier --write"
+    "prettify-updated": "pipe-git-updated --ext=css --ext=html --ext=js --ext=json --ext=md --ext=yaml --ext=yml -- prettier --write",
+    "pkg": "rm -rf node_modules && npm install && npx pkg . --out-path dist"
```

Comment:
Here we use `rm` and chain together commands via `&&`. Any suggestions on best practices / how to clean this up @medikoo ?

Reply:
I imagine that ideally this script should be setup as:

```
"pkg": "pkg bin/serverless.js"
```

Reply:
We have to remove `node_modules` because we include it in the binary via `assets` (also see your comment below). If we can get rid of `node_modules` we don't need to reset the project.

Reply:
I think we should assume here, that project is at state as if it was reset already (so have properly installed dependencies).

Why we do not assume that here? (in case of other scripts we rely on that assumption)

Reply:
I'm not sure who is building the binary at the end of the day. I can remove it.

Reply:
In the end I think we will prepare that in CI, same as npm publications